pkg/ip/cidr.go

```diff
@@ -71,6 +71,28 @@ func PrefixToIPNet(prefix netip.Prefix) *net.IPNet {
 	}
 }
 
+// IPNetsToPrefixes is a convenience helper for migrating from the older 'net.IPNet'
+// standard library type to the newer 'netip.Prefix' type. Use this to plug the old
+// type in existing code into the newer type.
+func IPNetsToPrefixes(ipNets []*net.IPNet) []*netip.Prefix {
```

Comment:
I feel reluctant to add these helpers back as they were removed by 929089ecb540a03a4ace8f15984ce5639caf7da8, but I see the reason to do so, we need them until pkg/ip gets rid of `net.IP` too,

Reply:
Can we use `netipx.FromStdIPNet` here? It's already a dependency we use in other packages.